app/components/publications/publication/case/info/publication-case-info-panel.hbs

```diff
@@ -191,13 +199,30 @@
                 </div>
               </div>
 
+              {{! wijze van publicatie }}
+              <div class="auk-o-grid-col-4 auk-u-mt-2">
+                <div class="auk-form-group">
+                  <Auk::Label>
+                    {{t "kind-of-publication"}}
+                  </Auk::Label>
+                  <PowerSelect
+                    @options={{this.publicationModes}}
+                    @onChange={{this.setPublicationMode}}
+                    @selected={{@publicationFlow.mode}}
+                    as |mode|
+                  >
+                    {{mode.label}}
+                  </PowerSelect>
+                </div>
+              </div>
```

Comment:
De velden kunnen in read- en edit mode best in dezelfde volgorde staan; met 4 op een rij.
Wat mij het meest logische lijkt:
Rij 1: Dringend | Pub nr. | Numac
Rij 2: Datum beslissing | datum ontvangst | Limit pub | Wijze pub